src/main/java/org/commcare/resources/model/Resource.java

```diff
@@ -111,7 +112,8 @@
     // Not sure if we want this persisted just yet...
     protected String parent;
 
-    private String descriptor;
+    protected String descriptor;
+    private boolean lazy;
```

Comment:
I think this means we're also going to need to bump the appdb database version in formplayer